kloppy/infra/serializers/tracking/signality.py

```diff
@@ -0,0 +1,267 @@
+import logging
+from datetime import timedelta, datetime
+import warnings
+from typing import List, Dict, NamedTuple, IO, Optional, Union
+from collections import Counter
+import numpy as np
```

Comment:
We don't need / want numpy as a dependency.

Reply:
We use numpy in the `attacking_directions_from_multi_frames()` function which I now moved to kloppy/domain/services/init.py. Is that okay? I assumed so since we were already using it in our skillcorner implementation.

Reply:
I fixed that yesterday in #375